doc/source/examples/multiphysics/capillary-wave/capillary-wave.rst

```diff
@@ -0,0 +1,379 @@
+================================
+Capillary Wave
+================================
+
+This example simulates the damping of a small amplitude capillary wave for different time-steps allowing to study the capillary time-step constraint. The problem is inspired by the test case of Denner *et al.* `[1] <https://doi.org/10.1016/j.jcp.2022.111128>`_
+
+
+--------
+Features
+--------
+
+- Solver: ``lethe-fluid`` 
+- Volume of fluid (VOF)
+- Unsteady problem handled by an adaptive BDF2 time-stepping scheme
+- Usage of Bash scripts to write, launch and postprocess multiple cases
+- Usage of Python scripts for postprocessing data
+
+
+---------------------------
+Files Used in This Example
+---------------------------
+
+- Analytical data: ``examples/multiphysics/capillary-wave/capillaryWaveData_rho_1_nu_5e-6_prosperetti.csv``
+- Analytical solution computation Python script: ``examples/multiphysics/capillary-wave/capillary-wave-prosperetti-solution.py``
+- Parameter file for case generation: ``examples/multiphysics/capillary-wave/capillary-wave.prm``
+- Parameter file for the :math:`\Delta t = 0.95\Delta t_\sigma` case: ``examples/multiphysics/capillary-wave/capillary-wave-TSM-0_95.prm``
+- Case generation and simulation launching Bash script: ``examples/multiphysics/capillary-wave/capillary-wave-time-step-sensitivity.sh``
+- Postprocessing Python script for a simple case: ``examples/multiphysics/capillary-wave/capillary-wave-postprocess.py``
+- Postprocessing Python script for generating a comparison figure: ``examples/multiphysics/capillary-wave/capillary-wave-combined.py``
+- Postprocessing Bash script: ``examples/multiphysics/capillary-wave/capillary-wave-time-step-sensitivity-postprocess.sh``
+- Python script for computing quantities of interest: ``examples/multiphysics/capillary-wave/capillary-wave-calculation.py``
```

Comment:
Here I would consider putting the path first in a sentence that introduces the list, and only the name of the file in the list.

Reply:
That's actually a great idea! I find it quit redundant too.